test/metabase/lib/js_test.cljs

```diff
@@ -398,3 +398,16 @@
            :day]
           (lib.js/expression-clause "time-interval" [(meta/field-metadata :products :created-at)
                                                      (lib.js/expression-clause "interval" [10 "day"] nil) "day"] nil)))))
+(deftest ^:parallel returned-columns-unique-names-test
```

Comment:
I think we could separate tests by a new line.

Reply:
True, that's my bad in editing the conflict. I'm not going to worry about it on the branch, though, since this is passing CI.